server/src/test/java/io/littlehorse/common/model/getable/core/wfrun/subnoderun/WorkflowEventRunTest.java

```diff
@@ -120,6 +120,27 @@ public void shouldIncrementWorkflowEventSequential() throws Exception {
         Assertions.assertThat(storedEvent).isNotNull();
     }
 
+    @Test
+    public void shouldIncrementWorkflowEventSequentialUsingCache() throws Exception {
+        WorkflowEventDefIdModel eventDef = new WorkflowEventDefIdModel("user-created");
+        ThrowEventNodeRunModel eventRun = new ThrowEventNodeRunModel(eventDef, testProcessorContext);
+        node.setSubNodeRun(eventRun);
```

Comment:
Could we use `nodeRun` rather than `node`? At first I was confused because I thought it was a `NodeModel` not a `NodeRunModel`